lib/evss/claims_service.rb

```diff
@@ -22,10 +24,14 @@ def submit_5103_waiver(claim_id)
       }.to_json
     end
 
+    def self.breakers_service
+      BaseService.create_breakers_service(name: 'EVSS/Claims', url: BASE_URL)
+    end
+
     protected
 
     def base_url
-      "#{ENV['EVSS_BASE_URL']}/wss-claims-services-web-3.0/rest"
+      BASE_URL
```

Comment:
It's just a little ugly because the method in the parent class has to be implemented as:

```
self.class::BASE_URL
```

But I guess seems worth doing.